etn-ccis/blui-react-component-library
# Convert Component playground(s) to use playground library

#### Changes proposed in this Pull Request:

- Convert following components' playground to use the library components from https://github.com/etn-ccis/blui-react-documentation-components
    - [x] ~~AppBar~~
    - [x] ~~ChannelValue~~
    - [x] Drawer
    - [x] DrawerHeader
    - [x] DrawerSubheader
    - [x] DrawerNavGroup
    - [x] DrawerNavItem
    - [x] DrawerRailItem
    - [x] DrawerFooter
    - [x] ~~EmptyState~~
    - [x] ~~Hero~~
    - [x] ~~InfoListItem~~
    - [x] ~~ListItemTag~~
    - [x] ScoreCard
    - [x] ThreeLiner
    - [x] ToolbarMenu
    - [x] UserMenu
- Remove unused files and redux state

#### Screenshots / Screen Recording (if applicable)

![image](https://github.com/etn-ccis/blui-react-component-library/assets/29152776/081916de-c45f-4164-9106-716e758970ce)

#### To Test:

- `yarn start:docs`
- go to updated playground pages and play around with the props
- See if anything is broken (other than missing copy button)

#### Any specific feedback you are looking for?

- What did we miss in the shared library?

> Known Issues (fixes in progress): 
> - Support for object-style props is a little weak (e.g., ChannelValueProps in Hero)
> - Large component snippets do not scroll
